privaterr.go

```diff
@@ -85,7 +85,7 @@ Fetch:
 
 	err := r.Data.Parse(text)
 	if err != nil {
-		return &ParseError{file, err.Error(), l}
+		return &ParseError{"", err.Error(), l}
```

Comment:
think `file` is/was quite useful here (and in the other `parse` functions)

Reply:
The file is added back to the error inside `ZoneParser.Next`. I agree that's not entirely clear, but I'm not sure where that could be documented. (`zp.setParseError` uses `zp.file` to fill in the `file` field).